lib/cocoapods/resolver.rb

```diff
@@ -448,6 +448,10 @@ def validate_platform(spec, target)
     def handle_resolver_error(error)
       message = error.message
       type = Informative
+      unless specs_updated?
+        specs_update_message = "\n * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`." \
+                               "\n   Note: as of CocoaPods 1.0, `pod repo update` does not happen on `pod install` by default."
```

Comment:
I think we can probably get rid of this note now, 1.0 shipped 3 years ago

Reply:
Done!